app/views/bookmarks/confirm_delete.html.erb

```diff
@@ -3,7 +3,7 @@
 
 <!--main content-->
 <%= form_for(@bookmark, :html => {:method => :delete, :class => "simple destroy"}) do |f| %>
-  <p class="caution notice"><%= ts('Are you sure you want to <strong><em>delete</em></strong> your bookmark of "%{bookmarkable}"?', :bookmarkable => @bookmark.bookmarkable.title).html_safe %></p>
+  <p class="caution notice"><%= ts('Are you sure you want to <strong><em>delete</em></strong> your bookmark of "%{bookmarkable}"?', :bookmarkable => (@bookmark.bookmarkable.nil? ? "This has been deleted, sorry!": @bookmark.bookmarkable.title)).html_safe %></p>
```

Comment:
This isn't on the bookmark itself, but rather a separate confirmation page for anyone who doesn't have JavaScript enabled, so it's relatively important to make sure users know what they're deleting (in case they have memory/cognitive issues, opened multiple delete windows, opened the window and walked away, etc). I'd suggest having two versions of the message, like:

```
<% if @bookmark.bookmarkable.nil? %>
  <%= ts('Are you sure you want to <strong><em>delete</em></strong> your bookmark of this deleted work?').html_safe %>
<% else %>
  <%= ts('Are you sure you want to <strong><em>delete</em></strong> your bookmark of "%{bookmarkable}"?', bookmarkable: @bookmark.bookmarkable.title).html_safe %>
<% end %>
```